app/routers/dayview.py

```diff
@@ -154,7 +172,7 @@ def get_all_day_events(
     day_end = day + timedelta(hours=24)
     for event in events:
         if is_all_day_event_in_day(event=event, day=day, day_end=day_end):
-            yield (event)
+            yield CurrentTimeAttributes(event)
```

Comment:
Why we wrap the event with this class? I'm having a bit of hard time understanding what is the purpose of the class and why/when we use it. It usually means that another reader would also have hard time understanding what this code does.

Reply:
I'm sorry! that was a mistake! :flushed: